packages/builder/src/internal/util/script-checkpoints.ts

```diff
@@ -0,0 +1,53 @@
+
```

Comment:
Please move this file to `src/lib/`

Reply:
Um.. ok. I also added `Deployer` class to this file.